### DISCUSSION TITLE
Need help with OAuth

### DISCUSSION THREAD

**Discussion by plusteamvn** [tweepy/tweepy#1660] [Questions] [answered]
Hi, I'm new to this and coding in general. 
I installed tweepy successfully. But when I tried to create OAuthHandler instance, it shows: 
```
import tweepy
>>> auth = tweepy.OAuthHandler(dA6ze_my_consumer_key, a6My_myconsumer_secret)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'dA6ze_my_consumer_key' is not defined
```
is it because I put my project in read-only mode? What did I do wrong?
UPDATE: I found the flaw, should have put quotation ""

**Comment by Harmon758** [comment, marked as answer]
As the error says, `dA6ze_my_consumer_key` isn't defined, which doesn't really have anything to do with Tweepy itself.
If `dA6ze_my_consumer_key` is supposed to represent the exact characters for your consumer key, rather than a variable that's set to your consumer key, then as your edit implies, you need to input it as a string. Otherwise, you're using it as a variable which hasn't been defined yet.
